From 94cdad75e68db80de67876a4c9350139b771cae3 Mon Sep 17 00:00:00 2001
From: zhaosiying12138 <neucszhaopeng2016@163.com>
Date: Mon, 24 Aug 2026 22:02:31 +0800
Subject: [PATCH 013/263] build: pin private TensorIR mode

---
 3rdparty/nvidia/tensor-ir | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/3rdparty/nvidia/tensor-ir b/3rdparty/nvidia/tensor-ir
index 3b5a8bebc00510b46a1e8b97229d06dd0d006215..ca8640e71cd5a95a041f60d8e18672c806f38e42 160000
--- a/3rdparty/nvidia/tensor-ir
+++ b/3rdparty/nvidia/tensor-ir
@@ -1 +1 @@
-Subproject commit 3b5a8bebc00510b46a1e8b97229d06dd0d006215
+Subproject commit ca8640e71cd5a95a041f60d8e18672c806f38e42
